services/http/api/tsconfig.json

```diff
@@ -16,6 +16,10 @@
             "interface/*": ["/lib/interface/index.ts"],
             "logging/*": ["/lib/logging/index.ts"],
             "requests/*": ["/lib/requests/index.ts"],
+            "models/*": ["/lib/models/index.ts"],
+            "client/*": ["/lib/models/client.ts"],
+            "measurement/*": ["/lib/models/measurement.ts"],
+            "resource/*": ["/lib/models/resource.ts"],
```

Comment:
how come we need client, measurement, resource if they are included in models? Just for clean import syntax?

Reply:
Yeah, but I ended up forgetting about them. I think I'll delete them.